CredentialManagerWebView/CredentialManagerHandler.kt

```diff
@@ -0,0 +1,51 @@
+package com.credmanwebtest.webhandler
+
+import android.app.Activity
+import android.util.Log
+import androidx.credentials.*
+import androidx.credentials.exceptions.*
+
+/**
+ * A class that encapsulates the credential manager object and provides simplified APIs for
+ * creating and retrieving public key credentials. For other types of credentials follow the
+ * documentation https://developer.android.com/training/sign-in/passkeys
+ */
+class CredentialManagerHandler(private val activity: Activity) {
+
+    private val mCredMan = CredentialManager.create(activity.applicationContext)
```

Comment:
@ksemenova i forgot this, we dont use this naming convention, this is an old style convention. You can keep this as credMan only. (as per the code standards)